README.md

```diff
@@ -28,19 +28,20 @@ This plugin can be installed from the [Update Center][3] (found at `Manage Jenki
 
 There are two ways to configure your plugin to submit data to Datadog:
 
-* **RECOMMENDED**: Using a DogStatsD server / Datadog Agent that acts as a forwarder between Jenkins and Datadog.
-  - Build Logs collection only works with a full Datadog Agent installed.
+* **RECOMMENDED**: Using a Datadog Agent that acts as a forwarder between Jenkins and Datadog.
+  - When using a DogStatsD server instead of a full Datadog Agent, only metrics and events are supported.
   - For metrics submitted from an external host, the Datadog Agent requires the following configuration for DogStatsD: `dogstatsd_non_local_traffic: true`. This can be configured using the `datadog.yaml` [configuration file][17].
```

Comment:
```suggestion
  - For data submitted from an external host, the Datadog Agent requires the following configuration: `dogstatsd_non_local_traffic: true` and `apm_non_local_traffic: true`. This can be configured using the `datadog.yaml` [configuration file][17].
```